test/hts-precompile/utils.js

```diff
@@ -525,6 +530,12 @@ class Utils {
     return accountInfo.accountId.toString();
   }
 
+  static async getContractInfo(evmAddress, client){
```

Comment:
Can you add a comment to note that the only reason we're using the SDK is to call the consensus node to confirm that there are no discrepancies between CN and MN.

Reply:
done